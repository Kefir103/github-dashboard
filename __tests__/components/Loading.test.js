import Loading from '../../src/client/components/Loading';
import React from 'react';
import renderer from 'react-test-renderer';

describe('<Loading />', () => {
    test('should be rendered correctly', () => {
        const component = renderer.create(<Loading />);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
