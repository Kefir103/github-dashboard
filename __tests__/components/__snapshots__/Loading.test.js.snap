// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> should be rendered correctly 1`] = `
<div
  className="lds-roller"
>
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
</div>
`;
